include/oxen/quic/format.hpp

```diff
@@ -44,7 +42,7 @@ namespace oxen::quic
         explicit buffer_printer(std::basic_string<T>&& buf) = delete;
 
         // Constructed from any type of span
-        template <oxenc::const_span_type T>
+        template <oxenc::bt_input_string T>
```

Comment:
This feels out of place, but I suppose this is only temporary until de-network PR is ready which replaces this (and the later similar changes) entirely.

Reply:
Yep, pretty much